docs/learn/learn-guides-polkadot-opengov.md

```diff
@@ -0,0 +1,64 @@
+---
+id: learn-guides-polkadot-opengov
+title: Polkadot OpenGov How-to Guides
+sidebar_label: OpenGov
+description: Advanced How-to Guides about Polkadot OpenGov.
+keywords: [opengov, polkadot opengov, referenda, cancel]
+slug: ../learn-guides-polkadot-opengov
+---
+
+import RPC from "./../../components/RPC-Connection";
+
+## Canceling Referenda using the Polkadot-JS UI
+
+:::info
+
+Anybody can cancel a referendum, given the referendum is ongoing (i.e., anywhere between Lead-in and
+voting/confirmation period). For more information about referenda timeline in Polkadot OpenGov see
+the [dedicated page](../learn/learn-polkadot-opengov.md#referenda-timeline).
+
+To successfully cancel a referendum through the track `20 / Referendum Canceller`, you will need to
+get a
+[specific approval and support](../maintain/maintain-guides-polkadot-opengov.md#referendum-canceller).
+
+:::
+
+To cancel a referenda, you need first to submit a preimage. Go to the
+[Polkadot-JS UI > Governace > Referenda](https://polkadot.js.org/apps/#/referenda) and click on the
+"Add Preimage" button. You must specify the specific call enacted if the proposal passes. In this
+case, we want to cancel a referendum, and we will need to specify the `referenda.cancel` extrinsic
+of referendum `249`.
+
+![cancel-referenda-preimage-creation](../assets/cancel-referenda-preimage-creation.png)
+
+This call will cancel the referendum and return the deposit. You can also decide to kill a
+referendum using the `referenda.kill` extrinsic. This will cancel the referendum and slash the
+deposit. Note that creating a preimage will reserve some funds proportionally to the amount of
+information stored within the preimage. In this case, the deposit was 1.33 KSM (see
+[Subscan](https://kusama.subscan.io/extrinsic/19143604-2)).
+
+Once a preimage is submitted, it can be checked under
+[Governance > Preimages](https://polkadot.js.org/apps/#/preimages).
+
+![cancel-referenda-preimage-check](../assets/cancel-referenda-preimage-check.png)
+
+You must copy the preimage to use it when you submit your proposal. To submit the proposal to cancel
+referendum 249, we need to go under
```

Comment:
Sometimes you use "you" and sometimes "we". I think you should stick with "you" for this.